test/entrypoint/config.yaml

```diff
@@ -0,0 +1,32 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        instances:
+          master:
+            roles: [roles.metrics-export]
+            roles_cfg:
+              roles.metrics-export:
+                http:
+                - listen: 8081
+                  endpoints:
+                  - path: /metrics
```

Comment:
We have used `/metrics/json` and `/metrics/prometheus` in examples before. Seems a bit less confusing

Reply:
Updated.